### PR TITLE
Determine local env

### PR DESCRIPTION
**Pull request by sktan**
Untested, but the logic should work.

### CODE REVIEW

**Comment by sktan** [comment]
@ArjenSchwarz let me know what you think or if this is not really what you were looking for.

**Comment by ArjenSchwarz** [comment]
Hey @sktan, it looks like what I want, but I've been too busy to test it. Hopefully tomorrow evening and otherwise it'll be the weekend. Don't worry, I won't forget about it :)

File: main.go [hunk 1]
```diff
@@ -45,7 +45,12 @@ var rawstartdate string
 var rawenddate string
 
 func init() {
-	flag.BoolVar(&local, "local", false, "Run locally")
+	task_path, task_exists := os.LookupEnv("LAMBDA_TASK_ROOT")
+	exec_path, exec_exists := os.LookupEnv("AWS_EXECUTION_ENV")
```
Comment (ArjenSchwarz) [inline main.go]:
```suggestion
	_, taskExists := os.LookupEnv("LAMBDA_TASK_ROOT")
	_, execExists := os.LookupEnv("AWS_EXECUTION_ENV")
```

File: main.go [hunk 2]
```diff
@@ -45,7 +45,12 @@ var rawstartdate string
...
+	local = true
+	if task_exists == true; exec_exists == true {
```
Comment (ArjenSchwarz) [inline main.go]:
```suggestion
	if taskExists || execExists {
```

**Comment by ArjenSchwarz** [comment]
Thanks @sktan, I've merged it with some small fixes.

**Pull request merged**